services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> Movie:
+    queryset = Movie.objects.all()
+    if genres_ids is not None:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids is not None:
+        queryset = queryset.filter(actors__id__in=actors_ids)
```

Comment:
You can simplify this code
```suggestion
    if not genres_ids:
        queryset = queryset.filter(genres__id__in=genres_ids)
    if not actors_ids:
        queryset = queryset.filter(actors__id__in=actors_ids)
```